vulcan/lib/client/jsx/components/vulcan_nav.jsx

```diff
@@ -116,8 +117,12 @@ const Halo = ({radius}) => {
 };
 
 function Logo() {
-  let {state} = useContext(VulcanContext);
-  const {calculating} = state;
+  const {state} = useContext(VulcanContext);
+  const {workflow, status} = state;
+
+  let calculating = workflow ? workflow.steps[0].some(
+    step => statusStringOfStepOrGroupedStep(step, workflow, status) == 'running'
+  ) : null;
```

Comment:
The error is this line -- step status should not be null, either undefined or StepStatus.  I'd make this undefined to fit the types.